.github/workflows/conda-build.yml

```diff
@@ -133,6 +133,31 @@ jobs:
         fi
         ./.scripts/run_osx_build.sh
 
+    # https://github.com/aktech/cirun-azure-resize-disk
+    - name: Resize all partitions to maximum
+      if: matrix.os == 'windows'
+      shell: pwsh
```

Comment:
This will need to move to smithy (probably with a config in `conda-forge.yml`), otherwise it'll get deleted on every rerender.

Reply:
https://github.com/conda-forge/conda-smithy/issues/2394